src/main/scala/com/spotify/spark/bigquery/package.scala

```diff
@@ -88,6 +88,15 @@ package object bigquery {
     }
 
     /**
+      * Set GCP pk12 key file.
```

Comment:
minor nitpicking: can you fix style so it's consistent with the rest? We're using the old JavaDoc style.